### PR TITLE
Upgrading wicket to 8.12, bootstrap to 4.6, popper to 1.16.1-lts, jquery to 3.6.0

### PR DESCRIPTION
**Pull request by mwhesse**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -55,28 +55,28 @@
...
 
         <!-- TODO -->
-        <commons-lang3.version>3.11</commons-lang3.version>
+        <commons-lang3.version>3.12.0</commons-lang3.version>
         <servlet-api.version>3.1.0</servlet-api.version>
         <javax.javaee-web-api.version>7.0</javax.javaee-web-api.version>
         <versions-maven-plugin.version>2.8.1</versions-maven-plugin.version>
         <doxia.version>1.0</doxia.version>
-        <modernizr.version>2.8.3</modernizr.version>
+        <modernizr.version>2.8.3-1</modernizr.version>
```
Comment (martin-g) [inline pom.xml]: I remember we needed to downgrade to 2.8.3 for some reason few years ago. I don't remember the exact reason now. Please revert this change.

File: pom.xml [hunk 2]
```diff
@@ -55,28 +55,28 @@
...
         <jetty.version>9.4.37.v20210219</jetty.version>
         <junit5.version>5.7.1</junit5.version>
         <google-closure.version>v20200406</google-closure.version>
         <contiperf.version>2.4.3</contiperf.version>
-        <joda-time.version>2.10.8</joda-time.version>
+        <joda-time.version>2.10.10</joda-time.version>
         <jquerypp.version>1.0.1</jquerypp.version>
-        <jquery.version>3.4.0</jquery.version>
+        <jquery.version>3.5.1</jquery.version>
```
Comment (martin-g) [inline pom.xml]: Why not 3.6.0 ?

File: pom.xml [hunk 3]
```diff
@@ -55,28 +55,28 @@
...
         <hamcrest.version>2.1</hamcrest.version>
         <mockito.version>3.6.0</mockito.version>
         <yuicompressor.version>2.4.8</yuicompressor.version>
         <maven-javadoc-plugin.version>3.2.0</maven-javadoc-plugin.version>
-        <wicket.version>8.9.0</wicket.version>
-        <wicketstuff.version>8.9.0</wicketstuff.version>
+        <wicket.version>8.11.0</wicket.version>
```
Comment (martin-g) [inline pom.xml]: 8.12 is already available

**Review by martin-g — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (mwhesse) [reply pom.xml]: Reverted to 2.8.3

File: pom.xml (hunk 2 above)
Comment (mwhesse) [reply pom.xml]: Upgraded to 3.6.0

File: pom.xml (hunk 3 above)
Comment (mwhesse) [reply pom.xml]: wicket and wicketstuff upgraded to 8.12, thanks.

**Review by martin-g — APPROVED** [review]

**Comment by martin-g** [comment]
Thank you, @mwhesse !
